README.md

```diff
@@ -1,13 +1,29 @@
-The IMOS Toolbox is mainly written in **Matlab** with a little bit of **Java** and aims at **converting oceanographic instrument files into quality controlled IMOS compliant NetCDF files**.
+# IMOS Toolbox
 
-Along with its source code, binaries are provided so that **it can also be used without Matlab**.
+The IMOS Toolbox aims at **converting oceanographic instrument files into quality-controlled IMOS compliant NetCDF files**. 
 
-It can connect to a deployment database in order to collect the relevant metadata attached to each data set. An MS-Access file is provided as a default deployment database but any type of database can be used. The Java code can use any JDBC API to query the deployment database. By default, [UCanAccess](http://ucanaccess.sourceforge.net/site.html) is used to query the MS-Access file.
+The toolbox process instruments deployed on moorings (**time series**) or during casts (**profiles**). The processing, including quality control (**QC**) of several oceanographic variables, can be done in **batch mode** or **interactive**.
 
-The toolbox can currently process instruments deployed on moorings (**time series**) or during casts (**profiles**).
+Finally, the package allows deployment metadata to be ingested into the files from MS-access databases, CSV files, or added manually through a graphical user interface (GUI). Manual **QC** is also possible.
 
-The processing of these files can be performed **interactively** with a graphical interface or in **batch mode**.
+See our [wiki page](https://github.com/aodn/imos-toolbox/wiki) page for more details.
 
-Please read the [wiki](https://github.com/aodn/imos-toolbox/wiki) for more information on how to **install** and **use** the toolbox. You can also read the [licence](https://raw.githubusercontent.com/aodn/imos-toolbox/master/license.txt) for copyright and licensing information.
+# Distribution/Requirements
 
-This project is designed and maintained by [ANMN](http://imos.org.au/facilities/nationalmooringnetwork/) and [AODN](http://imos.org.au/facilities/aodn/). It is licensed under the terms of the GNU GPLv3.
+The **stable** releases may be obtained [here](https://github.com/aodn/imos-toolbox/releases). The releases contain both the source code and binary applications (executables).
```

Comment:
Add information about installation with ongoing support and point to the install page of the wiki, you've removed this link and is important.

Reply:
done